src/entry_point.py

```diff
@@ -155,7 +155,11 @@ def _constructor_extract_conda_pkgs(prefix, max_workers=None):
             if pkg.endswith(ext):
                 fn = os.path.join(os.getcwd(), pkg)
                 flist.append(fn)
-    with tqdm.tqdm(total=len(flist), leave=False) as t:
+    if os.environ.get("CONDA_QUIET", "").lower() not in ("", "false", "0", "no"):
```

Comment:
`conda.auxlib.type_coercion.boolify` can do that in a more stable way

Reply:
Awesome, added!